webapp/components/integrations/components/installed_outgoing_webhook.jsx

```diff
@@ -69,28 +111,19 @@ export default class InstalledOutgoingWebhook extends React.Component {
 
     render() {
         const outgoingWebhook = this.props.outgoingWebhook;
-        const channel = ChannelStore.get(outgoingWebhook.channel_id);
+        const channel = this.props.channel;
         const filter = this.props.filter ? this.props.filter.toLowerCase() : '';
         const triggerWordsFull = 0;
         const triggerWordsStartsWith = 1;
 
-        if (!this.matchesFilter(outgoingWebhook, channel, filter)) {
+        if (!InstalledOutgoingWebhook.matchesFilter(outgoingWebhook, channel, filter)) {
             return null;
         }
 
-        let displayName;
-        if (outgoingWebhook.display_name) {
-            displayName = outgoingWebhook.display_name;
-        } else if (channel) {
-            displayName = channel.display_name;
-        } else {
-            displayName = (
-                <FormattedMessage
-                    id='installed_outgoing_webhooks.unknown_channel'
-                    defaultMessage='A Private Webhook'
-                />
-            );
-        }
+        const displayName = InstalledOutgoingWebhook.makeDisplayName({
```

Comment:
@hmhealey ya sure, I agree i don't think passing as an object makes much of sense here.